djangocms_moderation/forms.py

```diff
@@ -136,10 +136,13 @@ def __init__(self, user, *args, **kwargs):
     def set_collection_widget(self, request):
         related_modeladmin = admin.site._registry.get(ModerationCollection)
         dbfield = ModerationRequest._meta.get_field("collection")
+
+        remote_field = dbfield.rel if hasattr(dbfield, 'rel') else dbfield.remote_field
```

Comment:
```suggestion
        # Django 2.2 requires `remote_field` instead of `rel`. 
        remote_field = dbfield.rel if hasattr(dbfield, 'rel') else dbfield.remote_field
```